frontend/src/routes/(app)/report/[owner]/[report]/+page.svelte

```diff
@@ -82,6 +85,13 @@
 			<h1 class="text-5xl mr-6 text-grey-darkest">
 				{data.report.name}
 			</h1>
+			<LikeButton
+				on:like={() => zenoClient.likeReport(data.report.id)}
```

Comment:
Can this happen inside LikeButton?

Reply:
Can pass a function or do this. I think this is cleaner tbh.